src/cephadm/cephadmlib/templates/ceph.service.j2

```diff
@@ -8,9 +8,6 @@ Description=Ceph %i for {{fsid}}
 # configuration.
 After=network-online.target local-fs.target time-sync.target{% if has_docker_engine %} docker.service{% endif %}
 Wants=network-online.target local-fs.target time-sync.target
-{%- if has_docker_engine %}
-Requires=docker.service
```

Comment:
I think we'd be better to switch these to `Wants=` from `Requires=` rather than just removing them entirely. `Wants` will instruct systemd that it should bring up docker if it isn't already but without bringing down the ceph services the way Requires does when docker.service gets restarted.

Reply:
Sounds good, I didn't study Wants= - let me update the patch and test it out.

Reply:
Seems to work properly, thanks.